lib/osf-components/addon/components/registries/my-registrations-list/drafts/template.hbs

```diff
@@ -0,0 +1,25 @@
+<ContentPlaceholders as |placeholder|>
+    <PaginatedList::All
+        @modelName='draft-registration'
+        @usePlaceholders={{false}}
+        as |list|
+    >
+        <list.item as |draftRegistration|>
+            {{#if draftRegistration}}
+                <DraftRegistrationCard
+                    @draftRegistration={{draftRegistration}}
+                    @onDelete={{action list.doReload 1}}
```

Comment:
Not requesting a change, but does this reload to page 1 or to the page where the user was?

Reply:
This would reload to page 1.